.github/workflows/osx-installer.yml

```diff
@@ -97,25 +101,58 @@ jobs:
           die "Build failed"
 
           PATH=/usr/local/bin:$PATH \
-          make image ||
+          make GIT_BUILT_FROM_COMMIT=$GIT_BUILT_FROM_COMMIT image ||
           die "Build failed"
 
           mkdir osx-installer &&
           mv *.dmg disk-image/*.pkg osx-installer/
-      - name: Determine whether Scalar is included in the build
+      - name: Assign output variables
         run: |
+          # Determine whether Scalar is included in the build
           if (find ./stage/git-*/usr/local/git/bin/scalar); then
             echo "::set-output name=with_scalar::true"
           else
             echo "::set-output name=with_scalar::false"
           fi
+
+          # Set build commit hash
+          echo "::set-output name=commit_hash::$(git -C git rev-parse -q --verify HEAD 2>/dev/null)"
```

Comment:
Are we sure that this is the commit ID we want `git config --build-options` to show? I had expected something like this appended after line 85:

```sh
echo "::set-output name=commit_hash::$GIT_BUILT_FROM_COMMIT"
```

Reply:
This is mostly an attempt to keep the sources of the "actual" and "expected" commit hash values as separate as possible. By pulling the "expected" commit hash directly from `git rev-parse`, the validation later will confirm both that 1) the `dist` archive was built with the correct commit hash, and 2) that commit hash was properly embedded into the resulting installer.

Reply:
Fair enough!